calico_containers/tests/st/test_ipv6_multi_host.py

```diff
@@ -0,0 +1,50 @@
+from sh import ErrorReturnCode_1
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+from utils import retry_until_success
+
+
+class Ipv6MultiHostMainline(TestBase):
+    def test_ipv6_multi_host(self):
+        """
+        Run a mainline multi-host test with IPv6.
+
+        Almost identical in function to the vagrant coreOS demo.
+        """
+        host1 = DockerHost('host1')
+        host2 = DockerHost('host2')
+
+        ip1 = "fd80:24e2:f998:72d6::1:1"
+        ip2 = "fd80:24e2:f998:72d6::1:2"
+        ip3 = "fd80:24e2:f998:72d6::1:3"
+        ip4 = "fd80:24e2:f998:72d6::1:4"
+        ip5 = "fd80:24e2:f998:72d6::1:5"
+
+        workload1 = host1.create_workload("workload1", ip1, image="phusion/baseimage:0.9.16")
```

Comment:
Can you add a comment about the choice of image.  E.g. that `busybox` doesn't have IPv6 version of `ping`